### PR TITLE
Let's show on the overview page when the the site was last updated.

### PR DESCRIPTION
**Pull request by grandorf**
This is also useful for us when developing to see if latest build is actually whats showing on youseedk.github.io/dna

### CODE REVIEW

File: fractal-theme/scss/src/_default.scss [hunk 1]
```diff
@@ -2,8 +2,9 @@ html,
 body,
 .Prose,
 .Tree-item {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif,
-    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif,
```
Comment (kimblim) [inline fractal-theme/scss/src/_default.scss]: Why not just delete it? I still think this font-stack is superior for body text, but if you want to theme it otherwise, I think you should just delete it?

File: src/docs/02-UX and Design Guidelines/colors.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ A colour palette comprises primary and accent colours that can be used for illus
       <dt class="frctl-palette-title">{{@key}}</dt>
       {{#each this}}
         <dd class="frctl-color">
-          <div class="frctl-color-swatch" style="background-color: {{@this}}">
-             <code class="frctl-color-name" data-bg-color="{{@this}}">{{@this}}</code>
+          <div class="frctl-color-swatch" style="background-color: {{@this}};">
+             <span class="frctl-color-name" data-bg-color="{{@this}}">{{@this}}</span>
```
Comment (kimblim) [inline src/docs/02-UX and Design Guidelines/colors.md]: Why this change? It's showing a hex-code, which I think is "code"?

File: fractal-theme/scss/src/_default.scss (hunk 1 above)
Comment (grandorf) [reply fractal-theme/scss/src/_default.scss]: I like the idea. Don’t want to delete it just yet. But on windows the text is not pretty - I could not live with it :)
Maybe we want to use this again when we theme it

File: src/docs/02-UX and Design Guidelines/colors.md (hunk 2 above)
Comment (grandorf) [reply src/docs/02-UX and Design Guidelines/colors.md]: The font is ugly. If I use code I need to overwrite a lot of styling. It is just not that important for me so changed it to span.

Comment (kimblim) [reply src/docs/02-UX and Design Guidelines/colors.md]: I'm keeping the keys to the Dojo!
